packaging/tools/install-taostools.sh

```diff
@@ -72,17 +72,22 @@ function kill_process() {
 
 function install_main_path() {
     #create install main dir and all sub dir
-    [ ! -d ${install_main_dir}/bin ] && ${csudo} mkdir -p ${install_main_dir}/bin
+    [[ ! -d ${install_main_dir}/bin ]] && ${csudo} mkdir -p ${install_main_dir}/bin || :
 }
 
 function install_bin() {
     # Remove links
-    ${csudo} rm -f ${bin_link_dir}/taosdemo || :
-    ${csudo} rm -f ${bin_link_dir}/taosdump || :
+    ${csudo} rm -f ${bin_link_dir}/taosdemo         || :
+    ${csudo} rm -f ${bin_link_dir}/taosBenchmark    || :
+    ${csudo} rm -f ${bin_link_dir}/taosdump         || :
 
+    ${csudo} /usr/bin/install -c -m 755 ${script_dir}/bin/taosdump ${install_main_dir}/bin/taosdump
+    ${csudo} /usr/bin/install -c -m 755 ${script_dir}/bin/taosBenchmark ${install_main_dir}/bin/taosBenchmark
+    ${csudo} ln -sf ${install_main_dir}/bin/taosBenchmark ${install_main_dir}/bin/taosdemo
     #Make link
-    [ -x ${install_main_dir}/bin/taosdemo ] && ${csudo} ln -s ${install_main_dir}/bin/taosdemo ${bin_link_dir}/taosdemo          || :
-    [ -x ${install_main_dir}/bin/taosdump ] && ${csudo} ln -s ${install_main_dir}/bin/taosdump ${bin_link_dir}/taosdump          || :
+    [[ -x ${install_main_dir}/bin/taosdemo ]] && ${csudo} ln -s ${install_main_dir}/bin/taosdemo ${bin_link_dir}/taosdemo           || :
```

Comment:
should be `[[ -x ${install_main_dir}/bin/taosBenchmark ]] && ${csudo} ln -s ${install_main_dir}/bin/taosBenchmark ${bin_link_dir}/taosdemo || :` ?